### PR TITLE
enhanced css related filters

### PR DESCRIPTION
**Pull request by adon-at-work**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by adon-at-work** [comment]
@nera, i'm pushing my changes here. pls check. thanks.

File: src/xss-filters.js [hunk 1]
```diff
@@ -43,26 +47,164 @@ exports._getPrivFilters = function () {
...
 
     return (x = {
+        // turn invalid codePoints and that of non-characters to \uFFFD, and then fromCodePoint()
+        frCoPt: function(num) {
+            return !isFinite(num) ||            // `NaN`, `+Infinity`, or `-Infinity`
+                num <= 0 ||                     // NULL or not a valid Unicode code point
+                num > 0x10FFFF ||               // not a valid Unicode code point
+                // Math.floor(num) != num || 
+
+                (num >= 0x01 && num <= 0x08) ||
```
Comment (neraliu) [inline src/xss-filters.js]: @adon, would u comment the reference why u select those ranges for null replacement?

File: src/xss-filters.js [hunk 2]
```diff
@@ -43,26 +47,164 @@ exports._getPrivFilters = function () {
...
+                            : num === 0x96 ? '\u2013'  // EN DASH (–)
+                            : num === 0x97 ? '\u2014'  // EM DASH (—)
+                            : num === 0x98 ? '\u02DC'  // SMALL TILDE (˜)
+                            : num === 0x99 ? '\u2122'  // TRADE MARK SIGN (™)
+                            : num === 0x9A ? '\u0161'  // LATIN SMALL LETTER S WITH CARON (š)
+                            : num === 0x9B ? '\u203A'  // SINGLE RIGHT-POINTING ANGLE QUOTATION MARK (›)
+                            : num === 0x9C ? '\u0153'  // LATIN SMALL LIGATURE OE (œ)
+                            : num === 0x9E ? '\u017E'  // LATIN SMALL LETTER Z WITH CARON (ž)
+                            : num === 0x9F ? '\u0178'  // LATIN CAPITAL LETTER Y WITH DIAERESIS (Ÿ)
+                            : (num >= 0xD800 && num <= 0xDFFF) || num === 0x0D ? '\uFFFD'
```
Comment (neraliu) [inline src/xss-filters.js]: why the 0x0D will convert to \uFFFD?

Comment (adon-at-work) [inline src/xss-filters.js]: https://html.spec.whatwg.org/multipage/syntax.html#consume-a-character-reference

File: src/xss-filters.js (hunk 1 above)
Comment (adon-at-work) [inline src/xss-filters.js]: it's a simplified fromCodePoint():
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint#Polyfill

**Pull request merged**